src/tools/gophish_export.py

```diff
@@ -98,7 +98,9 @@ def export_targets(api, assessment_id):
 
             targets.append(target)
 
-    logging.info(f"{len(targets)} email targets found for assessment {assessment_id}.")
+    logging.info(
+        "%i email targets found for assessment %s.", len(targets), assessment_id
```

Comment:
I believe `%d` is now more standard than `%i`, even though `%i` still works:
```suggestion
        "%d email targets found for assessment %s.", len(targets), assessment_id
```

I don't see `%i` in the docs anymore: https://docs.python.org/3/library/string.html#format-specification-mini-language

Reply:
See aa93585